pages/[id].jsx

```diff
@@ -1,20 +1,57 @@
+import { useEffect, useState } from 'react';
 import { useRouter } from 'next/router';
 import Head from 'next/head';
 
+import apiService from '../services/apiService';
+import LoadingStatus, { loadingStates } from '../components/LoadingStatus';
+
 export default function DetailPage() {
+  const [teamMemberDetails, setTeamMemberDetails] = useState({});
+  const [loadingStatus, setLoadingStatus] = useState('EMPTY');
```

Comment:
Instead of using strings, it would be better to use the `loadingStates` that are set up. So this would be:

```suggestion
  const [loadingStatus, setLoadingStatus] = useState(loadingStates.EMPTY);
```